3rdParty/iresearch/core/store/directory.hpp

```diff
@@ -117,6 +117,9 @@ enum class IOAdvice : uint32_t {
   ///        explicitly required for MSVC2013
   ////////////////////////////////////////////////////////////////////////////
   READONCE_RANDOM = 6,
+
+  ///  file is opened for non-buffered reads
+  DIRECT_READ = 7
```

Comment:
Now `DIRECT_READ == SEQUENTIAL + RANDOM + READONCE` which is bad. In fact `IOAdvice` is a bit mask, i.e. I believe DIRECT_READ must be equal to `8`.